source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Yeah when I dove through MP when I added it to command-centre, gross was a pretty apt description